simpleocr/files.py

```diff
@@ -45,45 +32,152 @@ def write(self):
         write_boxfile(self.path, self.classes, self.segments)
 
 
-class ImageFile(object):
```

Comment:
I chose `ImageBuffer` in order to prevent naming problems when doing
```python
from PIL import Image
from simpleocr import ImageBuffer
```
and then I chose `GroundBuffer` to go along with the `ImageBuffer` name. It would of course be possible to do something like:
```python
from PIL import Image
from simpleocr import Image as OcrImage
```
or something similar instead. I don't mind which of the options is used, really, so it's your call ;) . In the mean-time, I will push my changes here and review them to check whether this draft is actually any good.